web/scripts/app.js

```diff
@@ -1297,8 +1311,18 @@ export class ComfyApp {
 			this.loadGraphData();
 		}
 
+		this.changing = false  // only used for administration for saved values for ctrl + Z
+
 		// Save current workflow automatically
-		setInterval(() => localStorage.setItem("workflow", JSON.stringify(this.graph.serialize())), 1000);
+		setInterval(function(){
+			const previous_workflow = localStorage.getItem("workflow")
+			const workflow = JSON.stringify(this.graph.serialize())
+			if(previous_workflow !== workflow){
+				if(!this.changing) localStorage.setItem("ctrlZ", previous_workflow);  // Save workflow for loading at ctrl + Z
+				this.changing = truep
```

Comment:
typo truep -> true

Reply:
Ok, for some reason it also worked fine with the typo so I didn't notice it XD. Thanks for pointing it out!